test/integration/decorators.ts

```diff
@@ -0,0 +1,75 @@
+/**
+ * @license
+ * Copyright (c) 2016 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+ */
+
+declare function fixture(id: string): HTMLElement;
+
+suite('TypeScript Decorators', function() {
+
+  suite('@customElement', function() {
+
+    test('defines an element', function() {
+      var testElement = fixture('test-element');
+      chai.assert.instanceOf(testElement, TestElement);
+    });
+
+  });
+
+  suite('@property', function() {
+
+    test('defines a property', function() {
+      var testElement = fixture('test-element') as TestElement;
```

Comment:
Maybe lets just add TestElement as the return type instead of doing a cast for almost all of the calls

Reply:
Well, `fixture` in the general case can return any type of element. It so happens here that we're only testing TestElement, but we could add some different fixtures too.

Really `fixture` should be defined in a WCT typings file, and it would be generic like this.

So I think I'll keep it?

Reply:
I would see it as a redefinition and narrowing down of the typings, as `fixture` returns HTMLElement, but you know better 😄 I am fine with keeping it as is of course.